src/datanode/src/datanode.rs

```diff
@@ -175,4 +175,18 @@ impl Datanode {
     pub fn get_instance(&self) -> InstanceRef {
         self.instance.clone()
     }
+
+    pub async fn shutdown_instance(&self) -> Result<()> {
```

Comment:
As the main purpose of this PR is adding close method to the region, I'd suggest revert the changes introduced by [387ada](https://github.com/GreptimeTeam/greptimedb/pull/970/commits/387ada6c54380cfac833a7415f508f00afd1f7e5)

Reply:
> As the main purpose of this PR is adding close method to the region, I'd suggest revert the changes introduced by [387ada](https://github.com/GreptimeTeam/greptimedb/pull/970/commits/387ada6c54380cfac833a7415f508f00afd1f7e5)

I see, let me handle it.